app/src/main/res/layout/fragment_aboutdiary1.xml

```diff
@@ -1,31 +1,133 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/frameLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@drawable/about_1"
+    android:background="@color/primary2"
     tools:context=".onboarding.FirstOnboardingFragment">
 
+    <!-- Surfing image -->
+    <ImageView
+        android:id="@+id/imgBackground"
+        android:layout_width="326dp"
+        android:layout_height="264dp"
+        android:layout_alignParentEnd="true"
+        android:layout_marginEnd="38dp"
+        android:background="@drawable/empty_list_background"
+        android:src="@drawable/surfing_pic" />
+
+    <!-- About Us Text -->
+    <TextView
+        android:id="@+id/about_us"
+        android:layout_width="318dp"
+        android:layout_height="79dp"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentTop="true"
+        android:gravity="center"
+        android:layout_marginLeft="51dp"
+        android:layout_marginTop="281dp"
+        android:fontFamily="@font/segoeprint"
+        android:text="About Us"
+        android:textColor="#000000"
+        android:textSize="55sp" />
+
+    <!-- Intro Text -->
+    <TextView
+        android:id="@+id/yada_is_yet"
+        android:layout_width="372dp"
+        android:layout_height="244dp"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentTop="true"
+        android:gravity="center"
+        android:layout_marginLeft="23dp"
+        android:layout_marginTop="366dp"
+        android:fontFamily="@font/segoeprint"
+        android:text="yada is `yet another diary app' created for travellers who want to write about their experiences."
+        android:textColor="#000000"
+        android:textSize="29sp"/>
+
+    <!-- Button to next page -->
     <ImageButton
         android:id="@+id/next1"
-        android:layout_width="wrap_content"
+        android:layout_width="67dp"
```

Comment:
We use sdp and ssp for scaling since they are flexible and can scale according to the device. Can you change 67dp to @dimen/_67sdp and all such occurrences?

Reply:
I changed all to sdp